lib/controllers/include/ActuatorController.h

```diff
@@ -20,18 +27,28 @@ class ActuatorController {
   virtual void overrideDegPerSecChecks() final;
   virtual void reinstateDegPerSecChecks() final;
```

Comment:
I feel like the wording is a bit ambiguous, maybe something like `activateDegPerSecChecks()` and `deactiveDegPerSecChecks()` would be more clear?